low-level/framework/base/sspl_constants.py

```diff
@@ -87,6 +87,8 @@
 DEFAULT_CLUSTER = "DEFAULT_CC01"
 CPU_PATH = "devices/system/cpu/"
 BLOCK_DIR = "/block/"
+DEFAULT_RECOMMENDATION = "Contact Seagate Support."
```

Comment:
There is already a variable defined for this at the bottom of this file. DEFAULT_ALERT_RECOMMENDATION. Please use that. You could also change the name to DEFAULT_HEALTH_RECOMMENDATION

Reply:
We can used it for both ALERTS and HEALTH right? should I keep it as DEFAULT_RECOMMENDATION only?

Reply:
Use/ rename it as DEFAULT_RECOMMENDATION, so it doesnt remain specific to Alerts